plugins/civo/api_key.go

```diff
@@ -0,0 +1,70 @@
+package civo
+
+import (
+	"context"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func APIKey() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.APIKey,
+		DocsURL:       sdk.URL("https://www.civo.com/docs/account/api-keys"), 
+		ManagementURL: sdk.URL("https://www.civo.com/api#authentication"), 
```

Comment:
Let's ensure that this field contains a user-specific page 😄 From what I see, this still links to the documentation.